draco/run.py

```diff
@@ -40,13 +43,25 @@ def run_clingo(
     :param models: Number of models to generate, defaults to 0 (meaning all models).
     :param topK: Whether to return the top K models. If false (default), the program
         will not optimize the output models.
+    :param constants: Dictionary of constants used in genenerator
```

Comment:
I wonder whether we should treat this as more general. What's happening is that we are passing in constants but how exactly they are used is irrelevant for `run`.